internal/buildmanifest/manifest.go

```diff
@@ -0,0 +1,66 @@
+package buildmanifest
+
+import "gopkg.in/yaml.v2"
+
+const (
+	// Filename of a build manifest.
```

Comment:
```suggestion
	// Filename of a build manifest relative to the package directory.
```

(assuming that is correct)